ts/state/ducks/onion.tsx

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import _, { forEach } from 'underscore';
+import { SnodePath, Snode } from '../../session/onions/index';
+
+export type OnionState = {
+  // nodes: Array<OnionPathNodeType>;
+  // path: SnodePath;
+  snodePath: SnodePath;
+};
+
+// const initialState: OnionState = {
+//   // nodes: new Array<OnionPathNodeType>(),
+//   nodes: new Array<Snode>(),
+// };
+
+const initialState = {
+  snodePath: {
+    path: new Array<Snode>(),
+    bad: false,
```

Comment:
might be good to start with a  bad path no? Or does redux starts too late and the first path is already built when we get here and so we don't get the event from the dispatch of new onion path?